examples/modern_art/src/main.rs

```diff
@@ -0,0 +1,142 @@
+use rand::{Rng, thread_rng};
+use crate::canvas::{Cursor, FillStyle, Geometry};
+use iced::widget::canvas::{Cache, Fill, Frame};
+use iced::widget::{canvas, Canvas};
+use iced::Settings;
+use iced::{
+    executor, Application, Color, Command, Element, Length, Point, Rectangle,
+    Renderer, Size, Theme,
+};
+use iced_graphics::gradient::Gradient;
+
```

Comment:
Shouldn't `Gradient` be available under the `iced::widget::canvas` since it's only use is in the canvas API? This extends to the `gradient` module as having access to `gradient::Builder` in application code will be needed.